src/types.ts

```diff
@@ -0,0 +1 @@
+export type ThemeColorOptions = "white" | "gray";
```

Comment:
Very tiny nit, but could you rename the type to `ThemeColorOption` (without the "s")? Even though this contains more than one option, the name of the _type_ should be singular, since it'll be used to describe a single object. For example, `const theme: ThemeColorOption = "white"`.

One way of thinking about it is that types and interfaces should be using the same kind of naming convention as classes, and you would give a class a singular name (e.g. `class Card`) even if you instantiate many objects from that class.

Reply:
Thanks for pointing this out!